OpenTrons/opentrons-api
# feat(protocol-designer): add tooltip to disabled modules

closes [RAUT-844](https://opentrons.atlassian.net/browse/RAUT-844)

# Overview

Certain combinations of staging area slots and modules will necessarily disable the addition of other modules. When these modules are disabled, their diagrams should have transparent backgrounds to show a gray enclosing flex, and hovering should render a tooltip describing why the module selection is disabled.

# Test Plan

1. Select all staging area slots, HS, TC, and TD modules, and trash bin _**OR**_ all staging area slots, 2 of HS, TC, and TD modules, and trash bin on file creation wizard
2. Observe grayed out background for remaining module or trash bin
3. Hover over disabled option and observe tool tip

<img width="742" alt="Screen Shot 2023-11-07 at 4 58 50 PM" src="https://github.com/Opentrons/opentrons/assets/47604184/42daf855-4e94-4149-9952-454392d44bf6">

# Changelog

- add tooltip to EquipmentOption rendered by ModulesAndOtherTile
- add corresponding tooltip text to translation assets
- add transparent background assets to modules diagram (may need to look for high-res)

# Review requests

@jerader per collaboration

# Risk assessment

low

[RAUT-844]: https://opentrons.atlassian.net/browse/RAUT-844?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
